src/yang/ietf-bgp.yang

```diff
@@ -858,6 +858,54 @@ module ietf-bgp {
             "Clear neighbors action.";
 
           input {
+	    choice operation {
```

Comment:
Not sure if this is indented correctly.

Reply:
The section had tabs.  Turns out my .vimrc wasn't being properly loaded.  I believe I've now fixed that.